### ISSUE TITLE
disable paid subscriptions

### ISSUE THREAD

**Issue by illori** [SimpleMachines/SMF#752]
https://github.com/SimpleMachines/SMF2.1/commit/3361365bd3d8f27278ebe7f6de54f9e4a428289d

after the above commit you can only enable paid subscriptions but not disable them again.

also why by default do you need to set the currency for paid subscriptions? can we just have a default set such as USD and let admin change it instead of telling them after enabling it that they have not set the currency?

**Comment by illori** [comment]
actually a clean install has the option to disable the paid subs, but an upgraded install that had core features does not let me disable paid subs.

**Comment by Arantor** [comment]
I don't see why there would be a problem with disabling it, works fine for me in terms of disabling. All the old core features stuff did was toggle $modSettings['paid_enabled'], and that's all the new setting in the paid subscriptions area does.

**Comment by illori** [comment]
![Alt text](http://imageshack.us/a/img580/2237/29he.png)

the disable does not show up at all, it is enabled as it was before from being a core feature. if i check with a clean install the disable option is present after enabling.

**Comment by Arantor** [comment]
Looks like you haven't updated Sources/ManagePaid.php, because the code branch in ManagePaid takes it in one of two directions - not enabled, shows only the enable option, or it shows that page... and the code is quite clear on this point.

```
        // These are all the default settings.
        $config_vars = array(
                array('check', 'paid_enabled'),
            '',
                array('select', 'paid_email', array(0 => $txt['paid_email_no'], 1 => $txt['paid_email_error'], 2 => $txt['paid_email_all']), 'subtext' => $txt['paid_email_desc']),
                array('text', 'paid_email_to', 'subtext' => $txt['paid_email_to_desc'], 'size' => 60),
```

There should be a line above the 'send notification emails' bit and there isn't.

**Comment by illori** [comment]
let me try again, maybe github messed up when i downloaded the zip from the main page, it had a different name from normal and is now back to normal... will reply again soon.

**Comment by illori** [comment]
the main issue seems solved with the new download of files. now all that remains is if we want to leave the currency as not setup or not.

**Comment by Arantor** [comment]
I wish I knew if there was a reason attached to that historically or not. I don't off hand know of any reason why we can't set one by default, but that doesn't mean there isn't one. I'm thinking about leaving it alone for now.
